exercises/complex-numbers/complex_numbers_test.rb

```diff
@@ -0,0 +1,207 @@
+require 'minitest/autorun'
+require_relative 'complex_numbers'
+
+# Common test data version: 1.0.0 '117d062'
+class ComplexNumbersTest < Minitest::Test
+  def test_imaginary_unit
+    # #skip
+    expected = ComplexNumber.new([-1, 0])
+    assert_equal expected, ComplexNumber.new([0, 1]) * ComplexNumber.new([0, 1])
+  end
```

Comment:
I really like how this test file has worked out 👍

Reply:
Thank you.
The user sees the test file and not the generator input, so I tried to make the test file as simple as possible.